ops/jenkins/global-pipeline-libraries/vars/awsEc2.groovy

```diff
@@ -3,11 +3,14 @@
 
 // Returns the AmiId for the given AMI and (Optional) git branch
 String getAmiId(String gitBranch, String amiName) {
-    command = "aws ec2 describe-images --owners self --filters \
-        ${gitBranch != "" ? "'Name=tag:Branch,Values=${gitBranch}'" : ""} \
-        'Name=name,Values=${amiName}' \
-        'Name=state,Values=available' --region us-east-1 --output json | \
-        jq -r '.Images | sort_by(.CreationDate) | last(.[]).ImageId'"
+    command = """
+set -x
```

Comment:
nit: This `set -x` can probably be removed.

Reply:
@malessi This was not something that was touched at all with these tickets. Do you want me to remove it anyway?